### DISCUSSION TITLE
Images are not visible (Migration from version 12 to 13)

### DISCUSSION THREAD

**Discussion by dimitardanailov** [vercel/next.js#44337] [Help]
Good Morning, 

I wish you happy holidays. 
I've been working to migrate website from version from version 12 to 13. 

Version: 12
https://fleetpal.io

Version: 13
https://fleetpal-landing-page-nr4aiia7w-fleetpal.vercel.app

My question: 
What's the reason the images to be broken. 

I've compared image string of one of the images:

version 13
```
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=400&q=75 400w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=640&q=75 640w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=750&q=75 750w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=828&q=75 828w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=1080&q=75 1080w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=1200&q=75 1200w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=1920&q=75 1920w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=2048&q=75 2048w, 
/?url=%2F_next%2Fstatic%2Fmedia%2FFleet_Unit.1a423460.webp&w=3840&q=75 3840w
```

version 12
```
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=400 400w, 
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=640 640w,
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=750 750w,
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=828 828w,
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=1080 1080w,
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=1200 1200w,
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=1920 1920w,
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=2048 2048w,
/_next/static/media/Fleet_Dashboard_Wigets.22071b04.webp?imwidth=3840 3840w
```

I'd like to share an issue which maybe related with the current ticket.

<img width="535" alt="Screenshot 2022-12-25 at 9 39 37 AM" src="https://user-images.githubusercontent.com/2182018/209460573-21d572bb-6f41-433e-98c3-836fff14b06b.png">

--- 

Update: 
I've create a new version: https://fleetpal-landing-page-aq68vlw8z-fleetpal.vercel.app/
The new version has a something like a temp patch. My patch is: The website uses as component -> `import Image from 'next/legacy/image'`

My goal is to migrate from the old version to the new version.

**Comment by adebiyial** [comment]
@dimitardanailov Why `next/legacy/image` and not `next/image` as in https://beta.nextjs.org/docs/optimizing/images?
